Jenkinsfile

```diff
@@ -51,8 +51,10 @@ pipeline {
             steps {
                 container('node') {
                     timeout(30) {
-                        sh "rm -rf ${YARN_CACHE_FOLDER}"
-                        sh "yarn --ignore-engines --unsafe-perm"
+                        withCredentials([string(credentialsId: "github-bot-token", variable: 'GITHUB_TOKEN')]) {
```

Comment:
I'm curious, why was this not needed before?

Reply:
It was just explained: To not hit the anonymous Github rate limits.

Reply:
the credentials allow us to retrieve the rigrep dependency from the github api in a more reliable way (50 vs 5000 request per hour). So the build works also without it, but fails if no requests are left.